code/modules/mob/living/silicon/robot/inventory.dm

```diff
@@ -118,8 +118,8 @@
 	var/obj/item/I = get_active_hand()
 	if(istype(I, /obj/item/gripper))
 		var/obj/item/gripper/G = I
-		G.drop_item_p(silent = 1)
-	return
+		G.drop_gripped_item(silent = TRUE)

```

Comment:
What if you were to modify bindings_robot.dm so that the drop key would make the gripper drop its item if it was holding something? I don't like overriding the `attack_self` capabilities, since I feel like that would easily bite us in the butt later if we want to add other attack-selfable things to grippers.

Reply:
Alright, I'll see about doing that.